test/lsp/lsp_completion.ml

```diff
@@ -76,7 +79,7 @@ let%expect_test "division-and-section-completion" =
        3           PROGRAM-ID . prog.
        4           DATA DIVISION.
     (line 1, character 24):
-    List of completions (1 entries): [DIVISION.]
+    List of completions (1 entries): [DIVISION.\n]
```

Comment:
This is the only occurrence of a completion from the middle of a word that I saw. In general what is the expected behaviors in these cases?

Reply:
The completion should start at the beginning of the word word (this is computed with the `range` function in `lsp_completion`.
I will add a test for that